docs/dev-guides/PERSONAL-ENVIRONMENTS.md

```diff
@@ -0,0 +1,523 @@
+dependencies:
+  - name: postgres
+    condition: postgres.enabled
+    version: 0.1.0
+    repository: https://broadinstitute.jfrog.io/broadinstitute/terra-helm-proxy
+```
+
+It's also possible to navigate the helm repo itself using the `helm` command:
+```shell script
+$ helm repo add terra-helm https://broadinstitute.github.io/terra-helm
+```
+```shell script
+$ helm repo update
+```
+```text
+...Successfully got an update from the "terra-helm" chart repository
+```
+
+Then just do 
+```shell script
+$ helm pull terra-helm/workspacemanager
+```
+and a .tgz file such as `workspacemanager-0.19.0.tgz` should be downloaded silently to the current directory.
+
+We don't need to make any changes to the `terra-helm` repo in order to set up a personal environment.
+
+### The Helmfile
+The helmfile allows specifying override values for a chart for a particular
+deployment, and the [terra-helmfile repo](https://github.com/broadinstitute/terra-helmfile)
+is the place providing and for editing those. This "gitOps" model
+is similar to the relationship between the private, Atlantis-enabled `terraform-ap-deployments` repo
+and the public `terraform-ap-modules` that we saw above.
+
+We don't actually need `helm` installed locally for this step, though it will be required later.
+
+
+Clone the [terra-helmfile](https://github.com/broadinstitute/terra-helmfile) and look at the file structure.
+```shell script
+$ tree -L 2
+```
+
+```
+.
+├── README.md
+├── argocd
+│   └── values
+├── bin
+│   ├── logging.sh
+│   └── render
+├── clusters.yaml
+├── docs
+│   └── NAMING.md
+├── envDefaults.yaml
+├── environments
+│   ├── live
+│   ├── live.yaml
+│   ├── personal
+│   ├── personal.yaml
+│   └── preview.yaml
+├── helmfile.yaml
+├── terra
+│   └── values
+├── versions
+│   ├── README.md
+│   ├── alpha.yaml
+│   ├── prod.yaml
+│   └── staging.yaml
+└── versions.yaml
+
+```
+You can see the main `helmfile.yaml`, an `environments` directory with subdirectories for `live` and `personal`, and a `terra` directory
+with subdirectories for each service:
+```shell script
+jaycarlton~/repos/terra-helmfile/terra (master) $ tree -L 2
+```
+
+```
+.
+└── values
+    ├── agora
+    ├── buffer
+    ├── buffer.yaml
+    ├── consent
+    ├── crljanitor
+    ├── crljanitor.yaml
+    ├── cromiam
+    ├── cromwell
+    ├── datarepo
+    ├── datarepomonitoring
+    ├── duos
+    ├── firecloudorch
+    ├── global
+    ├── global.yaml
+    ├── hydra
+    ├── icdemo
+    ├── identityconcentrator
+    ├── istioconfig
+    ├── istioconfig.yaml
+    ├── leonardo
+    ├── ontology
+    ├── poc
+    ├── poc.yaml
+    ├── rawls
+    ├── sam
+    ├── workspacemanager
+    └── workspacemanager.yaml
+```
+For this PR we'll edit all of
+* the helmfile itself, to make it aware of our personal enviornment
+* a personal.yaml file defining our environment's enabled services at a  high level.
+* several values files defining parameters for the individual services in the environment
+
+### Helmfile
```

Comment:
This section is almost identical in name to the prior section. Is there a clearer distinction between the two?

Reply:
renamed